### PR TITLE
add constructor method with username and password. #298

### PR DESCRIPTION
**Pull request by ndrlslz**
### Summary
add constructor method with username and password. #298

### CODE REVIEW

File: src/main/java/com/ibm/watson/developer_cloud/visual_recognition/v3/VisualRecognition.java [hunk 1]
```diff
@@ -90,6 +90,19 @@ public VisualRecognition(String versionDate) {
   }
 
   /**
+   * Instantiates a new Visual Recognition V2 service by username and password.
```
Comment (germanattanasio) [inline src/main/java/com/ibm/watson/developer_cloud/visual_recognition/v3/VisualRecognition.java]: `V2` should be `v3`

File: src/main/java/com/ibm/watson/developer_cloud/visual_recognition/v2_beta/VisualRecognition.java [hunk 2]
```diff
@@ -82,6 +82,18 @@ public VisualRecognition(String versionDate) {
     this.versionDate = versionDate;
   }
 
+  /**
+   * Instantiates a new Visual Recognition V2 service by username and password.
```
Comment (germanattanasio) [inline src/main/java/com/ibm/watson/developer_cloud/visual_recognition/v2_beta/VisualRecognition.java]: `v2` should be `v2-beta`

**Comment by germanattanasio** [comment]
I'm happy with merging this once we have the two issues mentioned above fixed.

Can you also add a `@see <link to service page>` and `@version <api version>` on each contructor?

For Tone Analyzer `v3-beta` it will be: 
```
/**
 * The IBM Watson Tone Analyzer service uses linguistic analysis to detect emotional tones, social
 * propensities, and writing styles in written communication. Then it offers suggestions to help the
 * writer improve their intended language tones.
 * 
 * @version v3-beta
 * @see <a href=
 *      "http://www.ibm.com/smarterplanet/us/en/ibmwatson/developercloud/tone-analyzer.html"> Tone
 *      Analyzer</a>
 */
```
you can also list the `version_date` so that users know what value they can use.

In Tone Analyzer v3 it will be `2016-05-19`

**Comment by ndrlslz** [comment]
@germanattanasio i did not catch your point about adding `@see` and `@version` on each constructor.

Because i can find theses tags in the beginning of class.

Is it necessary to add theses on the constructor repeatedly?

**Comment by germanattanasio** [comment]
good point @ndrlslz... I forgot about those tags being in the class...

**Pull request merged**